pkg/discovery/dtofactory/service_entity_dto_builder.go

```diff
@@ -76,7 +77,17 @@ func (builder *ServiceEntityDTOBuilder) BuildDTOs() []*proto.EntityDTO {
 
 		ebuilder := sdkbuilder.NewEntityDTOBuilder(proto.EntityDTO_SERVICE, id).
 			DisplayName(displayName)
-
+			// split service name and namespace to set as entity property
+		splitClusterServiceNameAndNameSpace := strings.Split(serviceName, "/")
+		clusterServiceNamespace, clusterServiceName := splitClusterServiceNameAndNameSpace[0], splitClusterServiceNameAndNameSpace[1]
+		// check the kube-system UID to set as entity property
+		kubeSystemUID, exists := builder.ClusterSummary.NamespaceUIDMap[kubeSystemPrefix]
```

Comment:
This property should be obtained before iterating over all services since it is the same for all services.